tests/ui/floating_point_hypot.fixed

```diff
@@ -0,0 +1,13 @@
+// run-rustfix
+#![warn(clippy::suboptimal_flops, clippy::imprecise_flops)]
+
+fn main() {
+    let x = 3f32;
+    let y = 4f32;
+    let _ = x.hypot(y);
+    let _ = (x + 1f32).hypot(y);
+    let _ = x.hypot(y);
+    // Cases where the lint shouldn't be applied
+    let _ = x.mul_add(x, y * y).sqrt();
```

Comment:
Why don't we want to include this case? Or is this just a TODO for the future?

Reply:
I've started trying to include this, but ended up giving up the added complexity. Left it as a TODO instead.

Reply:
Makes sense. Maybe add that in the comment here so it's clear it was just left out of the current impl